swagger/src/main/java/org/scalatra/swagger/runtime/annotations/ApiModelProperty.java

```diff
@@ -42,4 +42,14 @@
     * ordering, you should specify property order to keep models consistent across different VM implementations and versions.
     */
     int position() default 0;
+
+    /**
+     * Example value of the property. The format must be a valid json value strignyfied.
+     */
+    String example() default "";
```

Comment:
why would this be json?

Reply:
The best would have been to make this a `T` Type, but this is not a generic class, it's an annotation, and the type can't be anything. Any good Idea on how should I replace that type?

Note that in the end the json gets parsed printed directly in the swagger document, effectively allowing any type here.